README.md

```diff
@@ -702,15 +702,17 @@ __Query#dataSource('query', subQuery)__
 
 Set DimensionSpec. 
 
-Depending on arguments length creates default or extraction dimension spec.
+Give dimension spec as first argument.
```

Comment:
Change it to `If first argument is an object, then just use it as DimensionSpec.`?